core/azure-core/src/main/java/com/azure/core/Context.java

```diff
@@ -1,36 +1,38 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.core.http;
+package com.azure.core;
+
+import com.azure.core.http.HttpPipeline;
 
 import java.util.Optional;
 
 /**
- * {@code ContextData} offers a means of passing arbitrary data (key-value pairs) to {@link HttpPipeline}'s
+ * {@code Context} offers a means of passing arbitrary data (key-value pairs) to {@link HttpPipeline}'s
  * policy objects. Most applications do not need to pass arbitrary data to the pipeline and can pass
- * {@code ContextData.NONE} or {@code null}. Each context object is immutable.
- * The {@code addData(Object, Object)} method creates a new {@code ContextData} object that refers
+ * {@code Context.NONE} or {@code null}. Each context object is immutable.
+ * The {@code addData(Object, Object)} method creates a new {@code Context} object that refers
  * to its parent, forming a linked list.
  */
```

Comment:
Can you update the JavaDoc where necessary to no longer reference 'http' - because we want to use this for non-http things too (e.g. amqp pipelines).